### PR TITLE
Fix: Broken link in message input docs

### PR DESCRIPTION
**Pull request by khushal87**
## 🎯 Goal

This PR fixes the broken link on docusarus website as reported here - https://getstream.slack.com/archives/CP32GBDAA/p1642605506013300

## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] PR targets the `develop` branch
- [x] Commits follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) spec
- [ ] New code is covered by tests
- [ ] Screenshots added for visual changes
- [x] Documentation is updated

### CODE REVIEW

File: docusaurus/reactnative_versioned_docs/version-3.x.x/common-content/core-components/channel/props/input.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Component to render UI part of enclosed MessageInput component.
-Please read section in guides about [Custimizing Message Input](/reactnative/guides/message-input).
+Please read section in guides about [Customizing Message Input](/reactnative/guides/message-input-customization).
```
Comment (madsroskar) [inline docusaurus/reactnative_versioned_docs/version-3.x.x/common-content/core-components/channel/props/input.mdx:2]: Do we need to include this change in the v4.0.0 docs as well?

Comment (khushal87) [reply docusaurus/reactnative_versioned_docs/version-3.x.x/common-content/core-components/channel/props/input.mdx:2]: Umm, depends we can keep it against develop or v4. What's your call?

Comment (khushal87) [reply docusaurus/reactnative_versioned_docs/version-3.x.x/common-content/core-components/channel/props/input.mdx:2]: Already fixed for the `v4` version in the past.

Comment (madsroskar) [reply docusaurus/reactnative_versioned_docs/version-3.x.x/common-content/core-components/channel/props/input.mdx:2]: Woops, my bad :D

**Pull request merged**